drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -2357,6 +2358,18 @@ static int bcm2835_codec_open(struct file *file)
 		}
 		ctx->fh.ctrl_handler = hdl;
 		v4l2_ctrl_handler_setup(hdl);
+	} else if (dev->role == DECODE) {
+		v4l2_ctrl_handler_init(hdl, 1);
+
+		ctrl = v4l2_ctrl_new_std(hdl, &bcm2835_codec_ctrl_ops,
```

Comment:
Apologies if I missed this first time round, but ctrl is never used after this, so no need to define the new variable and assign to it.

Reply:
My bad. Left over from making V4L2_CID_MIN_BUFFERS_FOR_CAPTURE always return 1.